stdlib/public/Concurrency/Actor.cpp

```diff
@@ -582,16 +626,20 @@ static bool swift_task_isCurrentExecutorWithFlagsImpl(
     }
   }
 
-  if (options.contains(swift_task_is_current_executor_flag::HasIsIsolatingCurrentContext)) {
-    // We can invoke the 'isIsolatingCurrentContext' function.
+  // Invoke the 'isIsolatingCurrentContext' function if we can; If so, we can
+  // avoid calling the `checkIsolated` because their result will be the same.
+  if (options.contains(swift_task_is_current_executor_flag::UseIsIsolatingCurrentContext)) {
     SWIFT_TASK_DEBUG_LOG("executor checking: can call (%p).isIsolatingCurrentContext",
-    expectedExecutor.getIdentity());
+      expectedExecutor.getIdentity());
```

Comment:
Unnecessary indentation change

Reply:
It's fixing missing indentation tbh but I can avoid the change sure.